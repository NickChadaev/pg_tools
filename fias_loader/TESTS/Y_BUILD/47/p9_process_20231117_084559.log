2023-11-17 08:45:59 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_47 postgres /home/n.chadaev@abrr.local/Y_BUILD/47/stage_gar_c_47.csv 2023-11-13
--------------------------------------------------------
[08:45:59] ... Ленинградская обл., id_region: 46, Execute: /home/n.chadaev@abrr.local/Y_BUILD/47/stage_gar_c_47.csv, process_id:25345
[08:45:59] ... Start process
[08:45:59] ...  -- Version
[08:45:59] ...  -- Очистка от данных
[08:45:59] ...  -- Убираем индексы
[08:45:59] ...  -- Set UNLOGGED
[08:45:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[08:46:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_REESTR_OBJECTS_20231113_ad4e7a30-205e-4cdc-be7b-b1200d2f109b.XML
[09:13:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_ADDR_OBJ_20231113_3e2cfaff-1d8a-4e61-a4f5-14fd94d645de.XML
[09:14:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_HOUSES_20231113_315af324-40db-4625-b078-eab344c294b0.XML
[09:45:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_STEADS_20231113_3573b183-fb80-4864-802e-ce8110e88a02.XML
[09:49:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_APARTMENTS_20231113_d48cdbfa-05e4-4a8a-b9d3-b248aeeb4f90.XML
[10:01:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_ROOMS_20231113_554bb84c-56c5-4dcf-81e4-ab2b8a5fedbb.XML
[10:01:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_CARPLACES_20231113_73658f61-8b58-4229-b723-c7e9f41f1ebd.XML
[10:02:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_ADM_HIERARCHY_20231113_594e7336-b413-4d96-8eca-0977c50ab1c4.XML
[10:36:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_MUN_HIERARCHY_20231113_6830942a-c546-435b-bf08-a244f492bedc.XML
[11:06:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_CHANGE_HISTORY_20231113_9b9b0d77-6702-450a-8302-b4f799e5c3d6.XML
[12:30:21] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_ADDR_OBJ_DIVISION_20231113_7a221139-a6dc-4f9d-b055-90bb81edd6e3.XML
[12:30:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_ADDR_OBJ_PARAMS_20231113_bd6fbab9-0742-4015-9e25-ac760d4dc2ad.XML
[12:39:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_STEADS_PARAMS_20231113_296ca3f5-bd88-4b33-9030-e6f232eee24f.XML
[13:26:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_HOUSES_PARAMS_20231113_4d6e5b41-9c7d-41b3-84a9-c085a6104660.XML
[15:33:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_APARTMENTS_PARAMS_20231113_97da2e67-40b2-409b-a155-76a9eff84b51.XML
[15:51:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_ROOMS_PARAMS_20231113_3895081e-0da8-4046-8a7a-983941515b3c.XML
[15:51:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_CARPLACES_PARAMS_20231113_7280975b-60fc-4e6d-b128-ca228ec14806.XML
[15:51:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/47/AS_NORMATIVE_DOCS_20231113_eada6863-3416-46fe-9dc7-8bcaf56d6316.XML
[15:56:27] ...  -- Data finish
[15:56:27] ...  -- Set LOGGED
[15:57:26] ... Stop process
--------------------------------------------------------
