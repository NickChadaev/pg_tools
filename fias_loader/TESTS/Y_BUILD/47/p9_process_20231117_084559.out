 save_gar_version 
------------------
               82
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                82
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 2173226
gar_fias.as_addr_obj: 66420
gar_fias.as_houses: 2153448
gar_fias.as_steads: 316655
gar_fias.as_apartments: 902896
gar_fias.as_rooms: 1109
gar_fias.as_carplaces: 8970
gar_fias.as_adm_hierarchy: 2630551
gar_fias.as_mun_hierarchy: 2467024
gar_fias.as_change_history: 6627276
gar_fias.as_addr_obj_division: 310
gar_fias.as_addr_obj_params: 713189
gar_fias.as_steads_params: 3514091
gar_fias.as_houses_params: 10296454
gar_fias.as_apartments_params: 1369306
gar_fias.as_rooms_params: 1042
gar_fias.as_carplaces_params: 228
gar_fias.as_normative_docs: 397373
